src/components/Navbar/navbar.module.scss

```diff
@@ -23,6 +23,10 @@
   }
 }
 
+.userIcon {
```

Comment:
This class doesn't seem to be used anywhere.

Keep in mind that this is a CSS module and as such, the styles are not global but have to be explicitly bound to an element.

Reply:
Thanks for the feedback